src/ucs/datastruct/bitmap.h

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_BITMAP_H_
+#define UCS_BITMAP_H_
+
+#include <ucs/sys/compiler_def.h>
+#include <stdint.h>
+
+BEGIN_C_DECLS
+
+
+#define _UCS_BITMAP_BITS_IN_WORD (64)
```

Comment:
1. pls add typedef for ucs_bitmap_word_t
2. `#define UCS_BITMAP_BITS_IN_WORD  (sizeof(ucs_bitmap_word_t) * 8)`